etl/steps/data/garden/artificial_intelligence/2024-06-28/ai_incidents.py

```diff
@@ -0,0 +1,35 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Retrieve snapshot.
+    snap = paths.load_snapshot("ai_incidents.csv")
+
+    # Load data from snapshot.
+    tb = snap.read()
+
+    #
+    # Process data.
+    #
+
+    # Convert number of attendees to absolute numbers.
+    tb["Number of AI incidents"] = tb["Number of AI incidents"] * 1000
```

Comment:
I think this adjustment is not necessary, because the numbers become much larger compared to the previous version. See http://staging-site-ai-index/etl/wizard?page=chart-diff&chart_id=6927

Reply:
ahh nicely spotted! thanks a lot! :)